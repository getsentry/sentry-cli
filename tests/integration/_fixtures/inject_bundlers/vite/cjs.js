"use strict";function t(n,o){n(o)}function r(n,o){t(n,o)}function c(n,o){r(n,o)}c(function(o){throw new Error(o)},"boop");
//# sourceMappingURL=cjs.js.map
