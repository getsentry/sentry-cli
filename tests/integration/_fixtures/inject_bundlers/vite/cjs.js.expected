"use strict";
!function(){try{var e="undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{},n=(new e.Error).stack;n&&(e._sentryDebugIds=e._sentryDebugIds||{},e._sentryDebugIds[n]="c2fca958-c33e-5626-90a1-c6ce633b1f55")}catch(e){}}();
function t(n,o){n(o)}function r(n,o){t(n,o)}function c(n,o){r(n,o)}c(function(o){throw new Error(o)},"boop");
//# sourceMappingURL=cjs.js.map

//# debugId=c2fca958-c33e-5626-90a1-c6ce633b1f55
